TKGame/BackEnd/Art.cs

```diff
@@ -7,6 +7,8 @@ namespace TKGame.BackEnd
     static class Art
     {
         public static Texture2D PlayerTexture { get; private set; }
+        public static Texture2D PlayerLeftCrouch { get ; private set; }
+        public static Texture2D PlayerRightCrouch { get; private set; }
```

Comment:
Don't forget we can use the SpriteEffects enum to flip the sprite. Until we have a better player model that would have unique directional sprites we can minimize bloat